src/components/navigation-header/index.tsx

```diff
@@ -1,158 +1,223 @@
-import { ReactNode } from 'react'
+// Third-party imports
 import Link from 'next/link'
+
+// HashiCorp imports
+import HashiCorpLogo from '@hashicorp/mktg-logos/corporate/hashicorp/logomark/white.svg?include'
+import VaultLogo from '@hashicorp/mktg-logos/product/vault/primary-padding/colorwhite.svg?include'
+import WaypointLogo from '@hashicorp/mktg-logos/product/waypoint/primary-padding/colorwhite.svg?include'
+import InlineSvg from '@hashicorp/react-inline-svg'
+
+// Global imports
 import { productSlugsToNames } from '../../../config/products'
 import { ProductSlug } from 'types/products'
 import useCurrentPath from 'hooks/use-current-path'
-import { useCurrentProduct } from 'contexts'
+import { useCurrentProduct, useIsBetaProduct } from 'contexts'
 import HeaderSearchInput from 'components/header-search-input'
 import Text from 'components/text'
+
+// Local imports
 import { NavigationHeaderItem } from './types'
 import { NavigationHeaderDropdownMenu } from './components'
 import s from './navigation-header.module.css'
 
-const homePageNavItems = [
+/**
+ * Defines the navigation items for the main home page of the app. If this
+ * becomes authorable, it can be lifted into another area of the codebase.
+ */
+const HOME_PAGE_NAV_ITEMS = [
```

Comment:
I wonder if eventually it would make sense to extract this into a JSON file, similar to other config data.

Reply:
I think once we have more information about whether or not the header changes as we onboard more products, we'll be able to tell where this should live and if it should be authorable or not. The main goal of the src/data JSON files (I think that's what you're referencing here, let me know if not) is to prepare ourselves for authoring interfaces outside of the project (like Dato). Since all of the products rely on this not changing in a way that breaks each other, felt safest to not lift it into a JSON file yet.